views/appView.js

```diff
@@ -0,0 +1,31 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .submit': 'searchVideo'
+
+  },
+  initialize: function (){
+    console.log('view initialized')
+    // this.listenTo(this.model.get('videos'), 'reset', this.renderPage)
+  },
+
+  renderPage: function () {
+
+  },
+
+  searchVideo: function(){
+    console.log('search video function accessed')
+    var textData = $('#txtSearch').val();
+    this.model.get('videos').search = textData;
+    this.model.get('videos').fetch({
+      success: function (){
```

Comment:
See here is where you got so agonizingly close to a complete solution.  Either send a reset property to fetch and then re-render this view on the reset event or call re-render in this success callback.